spec/order_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "order"
+require "menu"
+
+describe Order do
+
+  describe '#select' do
+    it 'Selects one dish' do
+      subject.select_item('Egg Fried Rice')
+      expect(subject.print_selected_dishes).to eq('You have chosen to order: Egg Fried Rice')
+    end
+
+    it 'Selects multiple dishes' do
+      subject.select_item('Egg Fried Rice, Prawn Crackers')
+      expect(subject.print_selected_dishes).to eq('You have chosen to order: Egg Fried Rice, Prawn Crackers')
```

Comment:
This test failed when I ran it. I think it would be ok for `select_item` to take one dish name at a time and to call it multiple times. That would be less complex than parsing a comma separated list of dish names.

Please could you make changes so this failing test passes?